plugins/qtosgrave/qtosgviewer.h

```diff
@@ -84,6 +84,8 @@ class QtOSGViewer : public QMainWindow, public ViewerBase
     virtual GraphHandlePtr drawtrimesh(const float* ppoints, int stride, const int* pIndices, int numTriangles, const RaveVector<float>& color);
     virtual GraphHandlePtr drawtrimesh(const float* ppoints, int stride, const int* pIndices, int numTriangles, const boost::multi_array<float,2>& colors);
 
+    virtual GraphHandlePtr drawlabel(const std::string& label, const RaveVector<float>& voffset);
```

Comment:
I think instead of voffset variable name, better use worldPosition to make clear its a position in world space